test/parallel/test-net-server-connections-child-null.js

```diff
@@ -22,6 +22,11 @@ if (process.argv[2] === 'child') {
 
     // server.connections should start as 0
     assert.strictEqual(server.connections, 0);
+    server.getConnections(common.mustCall((err, conns) => {
+      // `conns` should start as 0
+      assert.ifError(err);
+      assert.strictEqual(conns, 0);
+    }));
```

Comment:
Given that this is an async operation, there's a risk of a possible race condition here. It's not triggered in this particular test currently but it's likely a good idea to guard against it still

Reply:
Thanks @jasnell for your review, and if you have any feedback please let me know